sqe-manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+    "dependencies": {
+        "SQE_DB_API": 0.0,
```

Comment:
Nice! We'll just need to keep this updated, and perhaps at a later point we can add a validation step (perhaps as part of the development server).

Reply:
Yeah, I was thinking of adding this into the auto install script so that it will read that manifest and rebuild the SQE_DB_API and/or the database in Data-files whenever a new version is needed.  I will see how things go, and add it when I find a little free time.